### DISCUSSION TITLE
40mp File Limit

### DISCUSSION THREAD

**Discussion by grafpaper1987** [pimcore/pimcore#17098] [General]
Hi there! I'm getting an error when I try to upload files >40MP. 

```
Image dimensions of file.jpg are too large.

Max size: 40 Megapixels

Suggestion: resize to 6000×4000 pixels or smaller.
```

Is there a way to adjust the maximum file size?

**Comment by natted** [comment]
@grafpaper1987 I haven't tried changing this but you can try update your config.yaml in the pimcore section

```
    pimcore:        
        image:
            max_pixels: 40000000
```

**Comment by grafpaper1987** [comment]
I added that to my config.yaml file but I'm still getting the same max size error. Am I doing this in the right place?

![image](https://github.com/pimcore/pimcore/assets/28054885/fcdb687f-5f68-4eb5-b642-88900e75f0bf)

**Reply by blankse** [reply]
@grafpaper1987 You changed the setting from 40 Megapixels to 10 Megapixels? I think you want to increase it, right?

**Reply by grafpaper1987** [reply]
Whoops. Good catch @blankse . I added a 0 to get to 100 Megapixels and still get the error that the max is 40 Megapixels. 

![image](https://github.com/pimcore/pimcore/assets/28054885/8c0174b4-3363-4f69-a8eb-9909aecf9d78)


See below for my updated config.yaml

![image](https://github.com/pimcore/pimcore/assets/28054885/e911b43d-117d-493e-a172-139dd0d30bef)

Any other thoughts here? Thanks for all the help so far.

**Reply by blankse** [reply]
@grafpaper1987 
1. Did you clear the symfony cache after changing the config.yaml? `php bin/console cache:clear`
2. Did you reload the entire admin interface after clearing the cache?
